src/concrete/concrete_choice.ml

```diff
@@ -15,6 +15,10 @@ let map v f =
   return (f v)
 [@@inline]
 
+exception ExplicitStop
+
+let stop = raise ExplicitStop
```

Comment:
This is raising the exception on module loading. You need something like: `let stop () = raise ExplicitStop` (but we should rather get rid of stop in the concrete case, see my other comment).

Reply:
Oopsie